lib/fbutil.js

```diff
@@ -16,6 +16,7 @@ exports.fbRef = function(path) {
 };
 
 exports.pathName = function(ref) {
+   if(typeof ref.parent === 'undefined') return ref.path;
```

Comment:
I don't see any usages except console.logs for this one, so more or less we can put anything here, so that the function does not throw exception and break the code flow. For my refBuilder ref.path was returning the path of monitor.